README.md

```diff
@@ -0,0 +1,113 @@
+## Requirements
+- Node.js v12.22.4
+- npm/npx v7.21.1
+- Docker v20.10.8
+- Cairo 0.5.1
+
+## Install Dependencies
+```
+npm install
+```
+
+## Use
+Make sure docker is running locally
+Generate the artifacts by compiling the contract using staknet-compile:
+### `starknet-compile`
+```
+npx hardhat starknet-compile
+```
+### `starknet-deploy`
+```
+npx hardhat starknet-deploy
```

Comment:
--starknet-network parameter is missing here if you want to use alpha network. Also, you should mention that user needs to prerun the devnet since I can see that localhost is set in the hardhat.config file.